### ISSUE TITLE
Make docs not working on travis

### ISSUE THREAD

**Issue by neerajprad** [pyro-ppl/pyro#242]
The sphinx doc build test is currently disabled after #212, because make docs was failing on travis due to changed references in one of our .rst files (it was passing locally as reported by @martinjankowiak). This needs to be fixed so that we can run the tests on all PRs before they get merged into dev.
